apps/client/src/components/CategoryItem.tsx

```diff
@@ -0,0 +1,307 @@
+import React, { useState } from 'react';
+
+interface CategoryItem {
+  id: number;
+  name: string;
+  description: string;
+  price: number;
+  stock: number;
+  pictures: string[]; // Array to store picture URLs
+}
+
+const CategoryItemsCrud: React.FC = () => {
+  const [categoryItems, setCategoryItems] = useState<CategoryItem[]>([
+    { id: 1, name: 'Laptop', description: 'High-performance laptop', price: 1000, stock: 10, pictures: Array.from({ length: 4 }, (_, i) => `https://picsum.photos/200/200?random=${i}`) },
```

Comment:
**Move `seed's` outside of the component and import them it makes component hard to read**